### PR TITLE
Fix memory leak when connecting wallet and make the spinner spin :)

### PR DESCRIPTION
**Pull request by alfetopito**
As subject says.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr109--dexreact.review.gnosisdev.com

File: src/components/Wallet.tsx [hunk 1]
```diff
@@ -46,7 +55,7 @@ const Wallet: React.FC<RouteComponentProps> = (props: WalletProps) => {
   if (loadingLabel) {
     content = (
       <>
-        <FontAwesomeIcon icon={faSpinner} />
+        <FontAwesomeIcon icon={faSpinner} spin={!!loadingLabel} />
```
Comment (anxolin) [inline src/components/Wallet.tsx]: I think it's good but the condition is redundant. You can just add:
```js
<FontAwesomeIcon icon={faSpinner} spin />
```

Comment (alfetopito) [reply src/components/Wallet.tsx]: That's true. Fixing it

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr109--dexreact.review.gnosisdev.com

**Pull request merged**